README.md

```diff
@@ -24,7 +24,9 @@ plugins: [
       // Set to true to display a bigger button
       tall: true, // default
       // Set to true to hide the text and display only a round P button
-      round: false // default
+      round: false, // default
+      // Set to true to add bookmark button to images
+      imageHover: false, // default
```

Comment:
This option should actually wrap the two others, because if it's set as `false` the `tall` and `round` won't apply (they are controlling the UI of the save button).

Then we could allow this kind of options:

### Default

If no options are passed at all, the script is loaded but doesn't affect the UI in any way.
```js
`gatsby-plugin-pinterest`
```

### With Save button

Users can opt in displaying the Pinterest `Save` button on image hover. This lets their visitors pin images to their Pinterest boards (this is probably the most common Pinit.js script use case).
```js
{
  resolve: `gatsby-plugin-pinterest`,
  options: {
    saveButton: true
  }
}
```

Note: I think that the `saveButton` option would be clearer than `imageHover`.

### With customized Save button

Some users might want to customize the way the Save button is rendered, and they can do it by passing additional sub-options:
```js
{
  resolve: `gatsby-plugin-pinterest`,
  options: {
    saveButton: {
      tall: true, // default
      round: false // default
    }
  }
}
```